common-tasks/dispvm.md

```diff
@@ -20,7 +20,30 @@ Once a DispVM has been created it will appear in Qubes VM Manager with the name
 
 See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why one would want to use a Disposable VM.
 
-Disposable VMs and Networking
+Disposable VMs and Networking (R4.0 and later)
+-----------------------------
+
+
+R4.0 introduces the concept of multiple disposable VM templates (R3.2 was limited to one).
+This allows for the creation of multiple differently configured disposable VMs that can be accessed from 
+the Applications menu. Even more types of DispVMs can be created on-the-fly on a per AppVM basis.
+As you can see, this is a very flexible and powerful system for managing your Disposable VMs.
+
+NetVM and firewall rules for Disposable VMs can be set as they can for a normal VM. 
+By default a DispVM will inherit the NetVM and firewall settings of the DispVM Template from which it is built. 
+Thus if an AppVM uses sys-net as its NetVM, but the default system DispVM uses sys-whonix,
+any DispVM launched from this AppVM will have sys-whonix as its NetVM.
+The default system wide DispVM template can be changed with `qubes-prefs default_dispvm`.
+You can change this behaviour for individual VMs: in the Application Menu, open Qube Settings
+for the VM in question and go to the "Advanced" tab. 
+Here you can edit the "Default DispVM" setting to specify which DispVM template will be used to launch DispVMs from that VM.
```

Comment:
Out of curiosity, can you set it to itself (i.e., make it so that all DispVMs launched from an AppVM are based on that AppVM)? If so, it might be worth mentioning this case.